webapp/components/Password/Strength.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="field">
+    <div class="password-strength-meter">
+      <div
+        class="password-strength-meter-inner"
+        :class="strengthClass"
+      />
+    </div>
+    <p class="help">
+      <span
+        v-if="pass"
+        :class="{ insecure: (passwordStrength < 3) }"
+      >
+        {{ $t('settings.security.change-password.passwordSecurity') }}:
+        <strong>{{ $t(`settings.security.change-password.passwordStrength${passwordStrength}`) }}</strong>
+      </span>
+      <span v-else>&nbsp;</span>
+    </p>
+  </div>
+</template>
+
+<script>
+import zxcvbn from 'zxcvbn'
+import { isEmpty } from 'lodash'
+
+export default {
+  name: 'PasswordMeter',
+  props: {
+    password: {
+      type: String,
+      required: true
+    }
+  },
+  data() {
+    return {
+      lastScore: 0,
+      pass: this.password || null
+    }
+  },
+  computed: {
+    strengthClass() {
+      return `strength-${this.passwordStrength}`
+    }
+  },
+  watch: {
+    /**
+     * passwordStrength is the score calculated by zxcvbn
+     * @return {Number} Password Strength Score
+     */
+    passwordStrength() {
```

Comment:
I moved this @ulfgebhardt because it was failing the linting
please see my commit message with a link to the SO post where I found the suggestion to move it to `watch`

Reply:
I Agree 100% - thanks for fixing this. As you can see i am still new to vue <3